### PR TITLE
Rounded borders utilities

### PR DESCRIPTION
**Pull request by gdostie**
Added utilities for rounded borders, also added a demo page for borders utilities.

image

### CODE REVIEW

File: scss/utility/border.scss [hunk 1]
```diff
@@ -31,3 +31,27 @@ $borders: border-top border-bottom border-left border-right;
 .mod-no-border {
   border: none;
 }
+
+@for $i from 1 through 10 {
```
Comment (MichelML) [inline scss/utility/border.scss]: add units https://stackoverflow.com/a/11275156

Comment (gdostie) [reply scss/utility/border.scss]: Where do I add the units ? `$i` is just a multiplicator of the `$border-radius` variable which already has `px` units.

**Comment by sroy3** [comment]
We usually use a border radius of 2. Is there a use for the other ones?

**Comment by gdostie** [comment]
@sroy3 yes, new UX mockups sometimes include a radius of `4px`, so instead of just adding this one I added some more just in case.

image

File: scss/utility/border.scss (hunk 1 above)
Comment (pochretien) [reply scss/utility/border.scss]: dont need, $border-radius already have px and your $i is only a multiplicator

Comment (MichelML) [reply scss/utility/border.scss]: sorry my mistake nvm @gdostie

**Pull request merged**